libs/platform/src/lib/components/form/combobox/combobox/combobox.component.html

```diff
@@ -0,0 +1,142 @@
+<ng-container *ngTemplateOutlet="mobile ? mobileTemplate : desktopTemplate"></ng-container>
+
+<ng-template #desktopTemplate>
+    <div cdkOverlayOrigin #selectTrigger="cdkOverlayOrigin">
+        <ng-container [ngTemplateOutlet]="controlTemplate"></ng-container>
+    </div>
+
+    <!-- Option panel using Material CDK overlay system -->
+    <ng-template
+        cdk-connected-overlay
+        [cdkConnectedOverlayOrigin]="selectTrigger"
+        [cdkConnectedOverlayOpen]="isOpen"
+        (overlayOutsideClick)="close($event)"
+        [cdkConnectedOverlayMinWidth]="minWidth">
+        <div class="fd-select-overlay fd-popover__body--no-arrow fd-popover__body--dropdown">
```

Comment:
Do you need `fd-select-overlay` here? Is something missing in combobox dedicated styling?

Reply:
fixed